master.css

```diff
@@ -350,7 +365,18 @@ label[for="sort"] {
     width: 260px;
 }
 .num-features {
-    position: absolute; top: -15px; right: 0px; font-size: 11px;
+    position: absolute;
+    top: -15px;
+    right: 0px;
+    font-size: 11px;
+    left: 0px;
+    background-color: #eee;
```

Comment:
(OK, I think I understand their purpose now, but IMHO, they should really be white boxes.)